src/core.c/Rat.pm6

```diff
@@ -79,8 +79,8 @@ sub DIVIDE_NUMBERS(
     )
 }
 
-# Initialize the $*RAT-UPGRADE-CLASS dynamic var so that it can be used
-PROCESS::<$RAT-UPGRADE-CLASS> = Num;
+# Initialize the $*RAT-UPGRADE-POLICY dynamic var so that it can be used
+PROCESS::<$RAT-UPGRADE-POLICY> = Num;
```

Comment:
Need to change it in the processkeys test also.

Reply:
Ah duh!